src/components/profile/afo-settings.js

```diff
@@ -1,20 +1,186 @@
-import React from "react";
+import React, {useState} from "react";
 import AfoNavbar from "../navbar/afo-navbar";
+import '../../styles/afo-profile.css';
+import userActions from "../../actions/user-actions";
+import {useDispatch} from "react-redux";
+import {useParams} from "react-router-dom";
 
 const Settings = () => {
+    const dispatch = useDispatch();
+    const {userId} = useParams();
+    const currentUser = userActions.findUserById(dispatch, userId);
+
+    const [imgUrl, setImageUrl] = useState(currentUser.pictureUrl);
+    const [username, setUsername] = useState(currentUser.username);
+    const [email, setEmail] = useState(currentUser.email);
+    const [password, setPassword] = useState(currentUser.password);
+    const [type, setType] = useState(currentUser.userType);
+    const [twitter, setTwitter] = useState(currentUser.twitter);
+    const [instagram, setInstagram] = useState(currentUser.instagram);
+    const [cardNum, setCardNum] = useState('');
+    const [bio, setBio] = useState(currentUser.bio);
+
+    const updateUser = () => {
+        //
+    };
+
+    const manageGroups = () => {
+        //
```

Comment:
Connected to manageGroups Button (In final version, it will open the group manager page)